### DISCUSSION TITLE
Editor Error in Serenity Form

### DISCUSSION THREAD

**Discussion by AHSOFFICE** [serenity-is/Serenity#7198] [General]
I want to make a parent-child like order and order-details in form like in northwind example. but when i write **VoucherDetailEditor** on child class this error shows up
![01](https://github.com/serenity-is/Serenity/assets/129037734/dbe64783-bb94-47ad-a74b-4fc9b963a8cd)
![02](https://github.com/serenity-is/Serenity/assets/129037734/4c41748b-7941-442b-b100-c4d8299b9d9b)
![Capture](https://github.com/serenity-is/Serenity/assets/129037734/f186a675-92ce-4559-8d22-18f781746b09)

**Comment by VictorTomaili** [comment]
You need to write VouncherDetailEditor in typescript, the attribute is generated from the typescript code. You should not write the attribute itself. Check the common features. https://github.com/serenity-is/common-features
